CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.3.0
+
+* Fix for Rails 5.0.2
```

Comment:
Yeah will do - I kind of figured this would be covered by the commit message, but no harm putting in more detail